inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd

```diff
@@ -164,21 +178,21 @@ plot_age_pyramid(vaccine_data_clean, age_group = "age_group", split_by = "sex")
 Accepting equal validity from self-reported and card-reported vaccination status
 ```{r vaccination coverage overall}
 
-suvery_ciprob(suvery_design, "vaccination_status_simple") %>%
+survey_ciprob(survey_design, "vaccination_status_simple") %>%
```

Comment:
you havent defined vaccination_status_simple yet anywhere?

Reply:
Earlier version of the template. Decided on a different angle and haven't update the remaining code yet